code/game/objects/items/weapons/stunbaton.dm

```diff
@@ -1,3 +1,5 @@
+#define HITTING_BUT_CAN_MISS -1 // Danke, old coders
```

Comment:
If the var did nothing by itself, then there's no need for the define, right?

Reply:
You saw nothing :^)